examples/routing_example/public/app.tsx

```diff
@@ -49,21 +49,21 @@ function RoutingExplorer({
                 listItems={[
                   {
                     label: 'IRouter API docs',
-                    href: 'https://github.com/elastic/kibana/blob/main/docs/development/core/server/kibana-plugin-core-server.irouter.md',
+                    href: 'https://github.com/elastic/kibana/blob/8.3/docs/development/core/server/kibana-plugin-core-server.irouter.md',
```

Comment:
nit: should we point to `main` and the API Docs?

Reply:
This file does no longer exist in main. I accept suggestions for replacement URLs, or I can use a permalink.

Reply:
The file's moved to `https://github.com/elastic/kibana/blob/main/src/core/packages/http/server/src/router/router.ts`.
We'd need to figure out when it moved and only backport the change to `main` to versions after the move.

Reply:
If we want to link to the actual generated docs, the new link should be https://docs.elastic.dev/kibana-dev-docs/api/kbn-core-http-server#:~:text=IRouter:

```suggestion
                    href: 'https://docs.elastic.dev/kibana-dev-docs/api/kbn-core-http-server#:~:text=IRouter',
```